policyengine_us/variables/gov/states/ne/tax/income/exemptions/ne_num_exemptions.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class ne_num_exemptions(Variable):
```

Comment:
We generally avoid `num` to avoid conflating with indexes (some exceptions may have slipped through the cracks)
```suggestion
class ne_count_exemptions(Variable):
```

Reply:
Eliminated variable, so done.